src/pages/Archive/Archive.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { useNotesContext } from "../../utils/contexts/NotesContext";
+import Note from "../../components/Note";
+function Notes() {
+  const { notes, saveNotes } = useNotesContext();
+
+  const renderNotes = () => {
+    return notes
+      .filter((f) => f.archived)
+      .map((note) => {
+        return <Note key={note.id} note={note} deleteNote={deleteNote} archiveNote={archiveNote} isArchived={true} />;
+      });
+  };
+
+  function deleteNote(noteId) {
+    const newList = notes.filter((f) => f.id !== noteId);
+    saveNotes(newList);
+  }
+
+  function archiveNote(noteId) {
+    const newList = [...notes];
+    const archived = newList.find((f) => f.id === noteId);
+    archived.archived = false;
+    saveNotes(newList);
+  }
```

Comment:
immutability is preferred, it makes the changes in the data predictable and easier to understand

```jsx
function archiveNote(noteId) {
  const archived = newList.find(({ id }) => id === noteId);
  const unarchived = {
    ...archived,
    archived: false,
  };
  const newList = [
    ...notes.filter(({ id }) => id === noteId),
    unarchived,
  ];

  saveNotes(newList);
}
```